app/Models/Champion.php

```diff
@@ -92,11 +153,41 @@ public static function find($name)
         return $champion;
     }
 
-    public static function addNew(string $name, int $hp, int $armor)
+    public static function addNew(string $name, int $hp,int $magicResist, int $armor, int $hpGrowth, int $armorGrowth, int $magicResistGrowth )
     {
-        $query = \App\Services\Db::get()->prepare("insert into lelek.champions (name, hp, armor) values ('$name', '$hp', '$armor')");
+        $query = \App\Services\Db::get()->prepare("insert into lelek.champions (name, hp, magicResist, armor, hpGrowth, armorGrowth, magicResistGrowth) values ('$name', '$hp','$magicResist', '$armor', '$hpGrowth',  '$armorGrowth', '$magicResistGrowth')");
         $query->execute();
         return static::find($name);
-}
+    }
+
+    public function levels()
+    {
+        $experience = 0;
+        for($i = 1; $i <=18; $i++)
+        {
+            $this->levelTreshold[$i][0] = $experience;
+            $experience = $experience + 99;
+            $this->levelTreshold[$i][1] = $experience;
+            $experience = $experience + 1;

```

Comment:
czyli dla level 1, mamy wartosc 0 oraz 99, po czym zwiekszamy to o 1 do 100 i lecimy druga petle, wiec kazdy level jest na 100 punktow?

a nie myslales wykorzystac tutaj np. ```floor($experience / 100) + 1``` jako wyznacznik levela? wtedy levels() byloby kompletnie niepotrzebne, a w checkLevel bys mial wtedy te podzialke.